pkg/analyser/wechat/wechat.go

```diff
@@ -93,15 +93,21 @@ func (w Wechat) GetAccountsAndTags(o *ir.Order, cfg *config.Config, target, prov
 		if r.Time != nil {
 			match, err = util.SplitFindTimeInterval(*r.Time, o.PayTime, match)
 			if err != nil {
-				log.Fatalf(err.Error())
+				log.Fatalf("Error in SplitFindTimeInterval: %v", err)
 			}
 		}
 		if r.TimestampRange != nil {
 			match, err = util.SplitFindTimeStampInterval(*r.TimestampRange, o.PayTime, match)
 			if err != nil {
-				log.Fatalf(err.Error())
+				log.Fatalf("Error in SplitFindTimeStampInterval: %v", err)
 			}
 		}
+		if r.MinPrice != nil && o.Money < *r.MinPrice {
+			match = false
+		}
+		if r.MaxPrice != nil && o.Money > *r.MaxPrice {
+			match = false
+		}
```

Comment:
I think we should also check the `MinPrice` and `MaxPrice`, make sure that `minPrice` is less than the `maxPrice`.

Reply:
SGTM, if the `minPrice > maxPrice`, then this rule is invalid, ignore it.